issuance/issuance.go

```diff
@@ -649,9 +657,15 @@ func (i *Issuer) Prepare(req *IssuanceRequest) ([]byte, *issuanceToken, error) {
 	}
 
 	if req.IncludeCTPoison {
+		if len(req.sctList) > 0 || len(req.precertDER) > 0 {
+			return nil, nil, fmt.Errorf("inconsistent request contains both precertificate and final certificate fields")
```

Comment:
I think that `i.Profile.requestValid(i.Clk, req)` above protects us from getting here, but it's better to be safe than sorry.

Just in case though, here's a test you may want to add. I put it in `TestIssueSCTList`, but there's probably a better location.
```
	_, _, err = signer.Prepare(&IssuanceRequest{
		PublicKey:       pk.Public(),
		Serial:          []byte{1, 2, 3, 4, 5, 6, 7, 8, 9},
		DNSNames:        []string{"example.com"},
		NotBefore:       fc.Now(),
		NotAfter:        fc.Now().Add(time.Hour - time.Second),
		IncludeCTPoison: true,
		precertDER:      []byte{6, 6, 6},
	})
	test.AssertError(t, err, "Invalid IssuanceRequest")
```

Reply:
Good find that this was already dealt with in requestValid. I removed the redundant check.